samples/boards/esp32/deep_sleep/src/main.c

```diff
@@ -81,6 +82,12 @@ int main(void)
 			ext_wakeup_pin_1, ext_wakeup_pin_2);
 	esp_sleep_enable_ext1_wakeup(ext_wakeup_pin_1_mask | ext_wakeup_pin_2_mask,
 			ESP_EXT1_WAKEUP_ANY_HIGH);
+
+	/* enable pull-down on ext1 pins to avoid random wake-ups */
+	rtc_gpio_pullup_dis(EXT_WAKEUP_PIN_1);
+	rtc_gpio_pulldown_en(EXT_WAKEUP_PIN_1);
+	rtc_gpio_pullup_dis(EXT_WAKEUP_PIN_2);
+	rtc_gpio_pulldown_en(EXT_WAKEUP_PIN_2);
```

Comment:
does this mean the kconfig doc for `EXAMPLE_EXT1_WAKEUP` should be updated? right now it is talking about what might happens with floating pins